### ISSUE TITLE
[24.10.0-rc1] Cannot install kernel modules

### ISSUE THREAD

**Issue by LaySoft** [openwrt/openwrt#17153]
### Describe the bug

I executed "opkg update", after I tried to install some kernel modules, but the following errors occured:
```
Collected errors:
 * opkg_install_cmd: Cannot install package kmod-nf-nathelper.
 * opkg_install_cmd: Cannot install package kmod-ipt-raw.
 * opkg_install_cmd: Cannot install package kmod-usb-storage.
 * opkg_install_cmd: Cannot install package kmod-usb-storage-uas.
 * opkg_install_cmd: Cannot install package kmod-fs-ext4.
```
### OpenWrt version

24.10.0-rc1

### OpenWrt release

24.10.0-rc1

### OpenWrt target/subtarget

mediatek/filogic

### Device

ASUS TUF-AX4200

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by LaySoft** [comment]
I installed this firmware:
`https://downloads.openwrt.org/releases/24.10.0-rc1/targets/mediatek/filogic/openwrt-24.10.0-rc1-mediatek-filogic-asus_tuf-ax4200-squashfs-sysupgrade.bin`

**Comment by namiltd** [comment]
Fixed: https://github.com/openwrt/openwrt/commit/ea17e958b9933aaf3e4a390073025aa226fe91b7
OpenWrt 23.10.0-rc2 released: 66e76aa94f0508d63a75fac2739a5bc0edb021ec
PR to close.
